security/SIG.md

```diff
@@ -6,22 +6,17 @@ Istio Security SIG and sub workgroups:
     *   Leads: Wencheng Lu (Google), Etai Lev-Ran (IBM)
     *   [Meeting notes](https://docs.google.com/document/d/12Xz8fCuql2tPL-cpdLXGGE44kYT4o9NB6mljSiS_01U)
     *   Scopes:
-        *   General Istio security topics across all sub workgroups, like Istio 0.3 security tasks.
-        *   Topics not covered by each sub workgroups, like ingress/egress, security threat model, etc.
+        *   General Istio security topics across all sub workgroups, like Istio general security tasks.
```

Comment:
should this file be somewhere in
https://github.com/istio/community instead maybe ?